src/main.rs

```diff
@@ -2,26 +2,36 @@ use cargo_v;
 use std::process::Command;
 use std::{env, fs, io};
 
+enum Operation {
+  Help,
+  Version(String),
+}
+
 fn main() {
   let mut args = env::args().skip(2);
 
-  let new_version = match args.next() {
-    Some(v) => v,
-    None => error_exit(
-      "failed to read the version: you must pass (patch, minor, major, etc.)",
-    ),
+  let operation = match args.next() {
+    Some(argument) => parse_cli(argument),
+    None => usage(),
   };
 
   let cargo_toml = match fs::read_to_string("./Cargo.toml") {
     Ok(toml) => toml,
     Err(e) => error_exit(&format!("failed to read Cargo.toml file: {}", e)),
   };
 
-  let new_version = match new_version.as_str() {
-    "patch" => cargo_v::VersionLabel::Patch,
-    "minor" => cargo_v::VersionLabel::Minor,
-    "major" => cargo_v::VersionLabel::Major,
-    v => cargo_v::VersionLabel::NumericVersion(String::from(v)),
+  if let Operation::Help = operation {
```

Comment:
Why not put this variant `Operation::Help` in `match` clause below?

Reply:
I tried to be explicit about handling the help operation, but I will move it to the match clause!